operator/v1/types_network.go

```diff
@@ -123,6 +123,35 @@ type NetworkMigration struct {
 	// changing the MTU for the default network will be rejected.
 	// +optional
 	MTU *MTUMigration `json:"mtu,omitempty"`
+
+	// features contains the features migration configuration. Set this to migrate
+	// feature configuration when changing the cluster default network provider.
+	// +optional
+	Features *FeaturesMigration `json:"features,omitempty"`
```

Comment:
What happens to feature migration when `Features` is not set?

Reply:
I added the description for this scenario.